setup.py

```diff
@@ -108,8 +108,13 @@
 
 def cython_extensions(package_names, module_names, extra_sources, include_dirs,
                       libraries, define_macros, build_with_cython=False):
+  # Set compiler directives linetrace argument only if we care about tracing;
```

Comment:
Huh. I was intending it to be an attributive noun, but, yeah, I guess one could read it the other way, too.